src/nostr-zap/zap-utils.ts

```
@@ -0,0 +1,330 @@
+  try {
+    const events = await (pool as any).list(relays, [
+      {
+        kinds: [9735],
+        '#p': [pubkey],
+      },
+    ]);
+
+        for (const event of events) {
+      const descriptionTag = event.tags.find((tag: string[]) => tag[0] === 'description');
+      if (descriptionTag && descriptionTag[1]) {
+        try {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)